src/libAtomVM/bif.c

```diff
@@ -75,6 +75,15 @@ term bif_erlang_byte_size_1(Context *ctx, int live, term arg1)
     return term_from_int32(term_binary_size(arg1));
 }
 
+term bif_erlang_bit_size_1(Context *ctx, int live, term arg1)
+{
+    UNUSED(live);
+
+    VALIDATE_VALUE(arg1, term_is_binary);
+
+    return term_from_int32(term_binary_size(arg1) * 8);
```

Comment:
most of the times we don't need to be that specific with int32, so we can just use term_from_int.

Reply:
I must have copied this from `bif_erlang_byte_size_1`  -- should that be changed, while I am at it?